app/customer.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+
+from app.extra import Car
+from app.extra import Point
+from app.shop import Shop
+
+
+class Customer:
+
+    @classmethod
+    def from_dict(cls, customers_data: dict) -> list:
+        return [Customer(
+                customer["name"],
+                customer["product_cart"],
+                Point(*customer["location"]),
+                customer["money"],
+                Car(customer["car"]["brand"],
+                    customer["car"]["fuel_consumption"]),
+                ) for customer in customers_data]

```

Comment:
Why classmethod? If you want to use classmethod you must use `cls` instead of `Shop`.
Or just use staticmethod